src/cascadia/TerminalControl/TermControl.cpp

```diff
@@ -1314,27 +1334,44 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         // std::stod will throw out_of_range expection if the input value is more than DBL_MAX
         try
         {
-            for (; std::getline(tokenStream, token, singleCharDelim) && (paddingPropIndex < thicknessArr.size()); paddingPropIndex++)
+            for (uint8_t paddingPropIndex = 0; std::getline(tokenStream, token, singleCharDelim) && (paddingPropIndex < 4); paddingPropIndex++)
             {
                 // std::stod internall calls wcstod which handles whitespace prefix (which is ignored)
                 //  & stops the scan when first char outside the range of radix is encountered
                 // We'll be permissive till the extent that stod function allows us to be by default
                 // Ex. a value like 100.3#535w2 will be read as 100.3, but ;df25 will fail
-                thicknessArr[paddingPropIndex] = std::stod(token, idx);
+                paddingArr.push_back(std::stod(token, idx));

```

Comment:
nit: idx is not used or needed here.  Consider removing this since we are modifying the code?